src/interfaces/CacheDriverInterface.ts

```diff
@@ -0,0 +1,11 @@
+import {PS2AlertsCensusCacheTypes} from './PS2AlertsCensusCacheTypes';
+
+export default interface CacheDriverInterface {
```

Comment:
There is an interface that called `CacheContract` available from ps2census. Implementations of that can be used by the managers.